Makefile

```diff
@@ -3,12 +3,12 @@ example:
 	tinygo build -o example/tiny_countvowels.wasm -target wasi ./example/countvowels
 	tinygo build -o example/tiny_http.wasm        -target wasi ./example/http
 
-	GOOS=wasip1 GOARCH=wasm go build -o example/std_countvowels.wasm ./example/countvowels
-	GOOS=wasip1 GOARCH=wasm go build -o example/std_http.wasm        ./example/http
+	GOOS=wasip1 GOARCH=wasm go build -tags std -o example/std_countvowels.wasm ./example/countvowels
+	GOOS=wasip1 GOARCH=wasm go build -tags std -o example/std_http.wasm        ./example/http
```

Comment:
added build tags to the files in `example/` so we can keep these single-path locations for the builds and conditionally compile the right code based on which compiler is used. 

the downside is we need to maintain two versions of the examples.

Reply:
@bhelx ^ this is one consequence but not a breaking change. 

the library does have breaking changes but nothing really extreme